procurify-dashboard/index.html

```diff
@@ -0,0 +1,317 @@
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <!-- <script src="main.js"></script> -->
+</head>
+<body>
+  <div class="wrapper">
+    <div class="header">
+    
+    </div>
+    <div class="sidebar">
+    
+    </div>
+    <div class="content-container">
+      <div class="row">
+        <div class="content-name">Dashboard</div>
+        <div class="panel tutorial-panel">
+          <div class="tutorial-content">
+            <h2>Getting Started</h2>
+            <span>Here are some of the awesome things you can do at Procurify.</span>
+          </div>
+          <ul class="tutorial-list row">
+            <li class="col"><div class="tutorial-step-group"><span class="dot filled"></span><p>Request</p></div></li>
+            <li class="col"><div class="tutorial-step-group"><span class="dot"></span><p>Approve</p></div></li>
+            <li class="col"><div class="tutorial-step-group"><span class="dot"></span><p>Purchase</p></div></li>
+            <li class="col"><div class="tutorial-step-group"><span class="dot"></span><p>Receive</p></div></li>
+          </ul>
+          <!-- <a href="" class="right btn tutorial-btn">Start Tutorial</a> -->
+        </div>
+      </div>
+      <div class="row pending-items-row">
+        <div class="col panel pending-item-tab">
+          <a href="">
+            <span class="pending-number">35</span>
+            <p class="pending-title">Pending Order Requests</p>
+          </a>
+        </div>
+        <div class="col panel pending-item-tab">
+            <a href="">
+              <span class="pending-number">22</span>
+              <p class="pending-title">Pending Expense Requests</p>
+            </a>
+        </div>
+        <div class="col panel pending-item-tab">
+            <a href="">
+              <span class="pending-number">14</span>
+              <p class="pending-title">Pending Travel Requests</p>
+            </a>
+        </div>
+        <div class="col panel pending-item-tab">
+            <a href="">
+              <span class="pending-number">54</span>
+              <p class="pending-title">Orders Pending Approval</p>
+            </a>
+        </div>
+        <div class="col panel pending-item-tab">
+            <a href="">
+              <span class="pending-number">15</span>
+              <p class="pending-title">Expenses Pending Approval</p>
+            </a>
+        </div>
+        <div class="col panel pending-item-tab">
+            <a href="">
+              <span class="pending-number">2</span>
+              <p class="pending-title">Travel Pending Approval</p>
+            </a>
+        </div>
+      </div>
+
+      <div class="row active-row">
+        <div class="col-7">
+          <div class="panel activity-panel">
+            <div class="activity-content">
```

Comment:
When naming child elements, it could be helpful to give as much context as possible when naming `.activity-panel` and its child could be `activity-panel_content`. What you did is completely acceptable, but as the app/site gets bigger it will become harder to manage!